### PR TITLE
Document configuration for ansible ad-hoc commands

### PR DESCRIPTION
**Pull request by DanLipsitt**
### Overview

This documents how to use Ansible ad-hoc commands with Vagrant.

### CODE REVIEW

**Comment by gildegoma** [comment]
@DanLipsitt Thank you for your interest in Vagrant and its Ansible provisioner. I am quite sorry to respond so late, especially because I am not in favour to accept this pull request and you certainly invested some precious time in building it.

Here are my arguments for not going forward with this request:

* Vagrant Ansible provisioner doesn't aim at providing **official** support for external/ad-hoc Ansible usage. The provisioner goal consists in providing a solid and stable integration within Vagrant context and commands (even with limitations (see https://github.com/mitchellh/vagrant/issues/3396#issuecomment-66905405 for more details about these guidelines).
* The ad-hoc command examples you propose will taint `~/.ssh/known_hosts`, while Vagrant Ansible provisioner is pretty careful regarding this point (by relying on `ANSIBLE_HOST_KEY_CHECKING=false` and ` ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null`). I mention this, because I don't want that the Vagrant documentation to become a source of confusion or additional support effort.

On the other hand the [**Using Vagrant and Ansible**](http://docs.ansible.com/ansible/latest/guide_vagrant.html) guide in the Ansible documentation, already has a section about **Running Ansible Manually** (to be faster than `vagrant provision`). 

This makes me realise that I intended for a long time to add a link to this guide in https://www.vagrantup.com/docs/provisioning/ansible.html#tips-and-tricks section, and... I am deeply astonished 😲 to notice that it isn't there. 

In conclusion: instead of the proposed **Ad-hoc commands** section, I'd propose to create a shorter one (e.g. **Running Ansible Manually**) with a "at your own risk" warning phrase and the link to http://docs.ansible.com/ansible/latest/guide_vagrant.html#running-ansible-manually for further details. @DanLipsitt Is it okay for you? (I can take care of it of course, but you're welcome if you want to adapt your PR)

**Comment by gildegoma** [comment]
Closing due to lack of reply and for above argumentation.
